src/trialsynth/base/util.py

```diff
@@ -146,3 +151,29 @@ def wrapper(*args, **kwargs):
         return method(*args, **kwargs)
 
     return wrapper
+
+
+@contextmanager
```

Comment:
This is to suppress scispacy logging: usually you can achieve this by creating a logger with the module name that creates the log messages
```
import logging
spacy_logger = logging.getLogger('spacy')
spacy_logger.setLevel(logging.ERROR)
```